ext/intl/dateformat/dateformat.stub.php

```diff
@@ -118,6 +118,12 @@ public static function formatObject($datetime, $format = null, ?string $locale =
      */
     public function parse(string $string, &$offset = null) {}
 
+    /**
+     * @return string|false
+     * @alias datefmt_get_best_pattern
+     */
+    public static function getBestPattern(string $skeleton, ?string $locale = null) {}
```

Comment:
New methods can have a return type declaration. Return types are not declared for existing methods only because of BC purposes. See https://wiki.php.net/rfc/internal_method_return_types